src/i18n/locales/en.ts

```diff
@@ -105,7 +105,33 @@ export const en = {
     add: 'Add',
     add_other_source_type: 'Add other source type',
     configure_outputs: 'Configure Outputs',
-    manage_multiviewers: 'Manage multiviewers'
+    manage_multiviewers: 'Manage multiviewers',
```

Comment:
A small thing, but for consistency, the plural use of 'Multiview' that we use in the project is 'Multiviews'.
The same comment applies in the Swedish file.